csm/StorageMan.py

```diff
@@ -0,0 +1,44 @@
+import queue
+# import cv2
+import numpy as np
+import os
+from os.path import isfile, join
+
+class StorageMan():
+
+    def __init__(self):
+        self.frames = []
+        self.recvFrameQ = queue.Queue(maxsize=1)
+        self.compileQ = queue.Queue(maxsize=1)
+        self.width = -1
+        self.height = -1
+
+    def appendFrame(self, frame, frame_width, frame_height):
+        self.recvFrameQ.put((frame, frame_width, frame_height))
+
+    def compile(self, outputPath, fps):
+        self.compileQ.put((outputPath, fps))
+
+    def launch(self):
+        """
+        Launches the Storage Manager. Loop will repeatedly listen to concurrent
+        frame insertions. When a compile command is given, 
+        """
+        while self.compileQ.empty():
+            if not self.recvFrameQ.empty():
+                frame, self.width, self.height = self.recvFrameQ.get()
+                self.frames.append(frame)
+                print("StorageMan   : Frame received and stored.")
+
+        opPath = self.compileInternal()
+        print("StorageMan   : Frames compiled and output saved at: %s" % opPath)
+
+    def compileInternal(self):
+        outputPath, fps = self.compileQ.get()
+        # output = cv2.VideoWriter(outputPath, cv2.VideoWriter_fourcc(*'DIVX'), fps, (width, height))
```

Comment:
Can  you add a comment here about the current state of things (i.e. is it a WIP or like a TODO and then things you need to figure out)?